packages/dart/npt_flutter/pubspec.yaml

```diff
@@ -83,6 +83,7 @@ dev_dependencies:
     sdk: flutter
 
 dependency_overrides:
+  meta: 1.15.0
```

Comment:
at_commons use in no_ports core from path requires meta 1.16.0 but flutter_tests and flutter_localizations requires meta 1.15.0. Meta is pinned to 1.15.0 in the flutter SDK.